artifactory/commands/transferfiles/utils.go

```diff
@@ -24,9 +24,8 @@ import (
 const (
 	waitTimeBetweenChunkStatusSeconds            = 3
 	waitTimeBetweenThreadsUpdateSeconds          = 20
-	assumeProducerConsumerDoneWhenIdleForSeconds = 15
+	assumeProducerConsumerDoneWhenIdleForSeconds = 7
```

Comment:
I suggest we do not change this value. We will implement a future solutuon that will avoid this wait time altogether.